reference/unity_to_godot.rst

```diff
@@ -0,0 +1,177 @@
+.. _unity3D_to_godot:
+
+..    references : 
+..    https://wiki.unrealengine.com/Unity3D_Developer's_Guide_to_Unreal_Engine_4
+..    https://docs.unrealengine.com/latest/INT/GettingStarted/FromUnity/
+
+From Unity3D to Godot Engine
+============================
+
+This guide provides an overview of Godot Engine from the viewpoint of a Unity user, and aims to help you migrate your existing Unity experience into the world of Godot.
+
+Differences
+-----------
+
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+|                   | Unity                                                                             | Godot                                                                                                          |
++===================+===================================================================================+================================================================================================================+
+| License           | proprietary                                                                       | MIT License                                                                                                    |
```

Comment:
I wold explain license extents a bit more, like:

Proprietary, closed, free license with revenue caps and usage restrictions
vs
MIT License, free and fully open souce without any restriction